specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/gallery.json

```diff
@@ -98,6 +98,9 @@
         "x-ms-examples": {
           "Create or update a simple gallery.": {
             "$ref": "./examples/CreateOrUpdateASimpleGallery.json"
+          },
+          "Create or update a simple gallery with sharing profile.": {
+            "$ref": "./examples/CreateOrUpdateASimpleGalleryWithSharingProfile.json"
```

Comment:
what about the changes to PutGallery?  The permissions can only be modified in PutGallery (from private to groups)

Reply:
This is for the PutGallery under the operation 'put'. Do you mean I need more examples or add more examples to patch?